module3/src/main/java/com/epam/esm/module3/controller/localization/LanguageConfig.java

```diff
@@ -1,18 +1,20 @@
-package com.epam.esm.config.localization;
+package com.epam.esm.module3.controller.localization;
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.support.ResourceBundleMessageSource;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import org.springframework.web.servlet.i18n.AcceptHeaderLocaleResolver;
+
 import javax.servlet.http.HttpServletRequest;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
 
 /**
- * Class contains spring configuration for module to work with localization.
+ * Class contains spring configuration to work with localization.
  */
+
```

Comment:
Where is access modificator for List<Locale> ? Also apply upper case to class fields only if they are static final, according to convension.